src/components/Modals/Onboarding/Onboarding.scss

```diff
@@ -313,7 +313,6 @@ input {
     border-top: 1px solid #e5e5e5;
     box-sizing: border-box;
     box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
```

Comment:
May want to check in with Joanna on the shadow CSS - we notice there were some inconsistencies across CoursePlan with this so we might as well fix them here